src/CloudFoundry.VisualStudio/ProjectPush/PublishProfile.cs

```diff
@@ -477,15 +404,14 @@ public void Save()
 
             if (this.environment.IsProjectWebsite)
             {
-                SaveWebsiteProject();
+                this.SaveWebsiteProject();
             }
         }
 
-        public event PropertyChangedEventHandler PropertyChanged;
-
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate", Justification = "MVVM model")]
```

Comment:
The Justification here should be: "This is a helper method that raises an event"